### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by JamesOkunlade**
Pull request tutorial for Olusola

### CODE REVIEW

File: spce.html [hunk 1]
```diff
@@ -23,9 +23,7 @@ <h1>Spacebook - the trusted way to experience short lets and serviced apartments
                 </div>
                 <div class="right-side">
                     <form action="">
-                        <div class="area">
-                            <input type="text" placeholder="Area e.g (yaba,lagos)"><br>
-                        <div>
```
Comment (olusolaige) [inline spce.html:28]: Why did you remove this?

**Review by olusolaige — CHANGES_REQUESTED** [review]
Please fix requested changes

File: spce.html (hunk 1 above)
Comment (JamesOkunlade) [reply spce.html:28]: Good point for the inline comment tutorial. It's just basic refactoring.

**Review by olusolaige — APPROVED** [review]
Good job 👍 
Go ahead and merge PR
